.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -374,7 +375,7 @@ jobs:
         auditwheel repair dist/*.whl -w ./wheel --no-update-tags --exclude libopenblasp-r0-23e5df77.3.21.dev.so
 
     - name: Upload Wheel Artifact
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
```

Comment:
Just a note here to share via slack.

Reply:
Yes v4 should be supported now since we upgraded to manylinux_2_28 :)